buffother.lic

```diff
@@ -0,0 +1,46 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#buffother
+=end
+custom_require.call(%w(common common-arcana events spellmonitor drinfomon common-travel))
+class Buffother
+  include DRC
+  include DRCA
+  include DRCT
+
+  def initialize
+    arg_definitions = [
+      [
+        { name: 'recipient', regex: /\w+/, description: 'Person to buff' },
+        { name: 'waggle', regex: /\w+/, description: 'Waggle set to cast' },
+        { name: 'room', regex: /\d+/, optional: true, description: 'Room to move to' }
+      ]
+    ]
+    args = parse_args(arg_definitions)
+    @settings = get_settings
```

Comment:
💡 I'd prefer to see these as non-instance vars and pass them as function args. What do you think?

Reply:
While I don't actually understand the merits one way or another (noob here), I've made those adjustments.